buzzmoon-frontend/buzzmoon-vite/src/Components/QuestionSpeaker/QuestionSpeaker.jsx

```diff
@@ -25,18 +30,23 @@ export default function QuestionSpeaker(props) {
 
   const buzz = () => {
     audioRef.current.pause();
+    audioRef.current.muted = true;
     props.setBuzzTimings(
       { ...props.buzzTimings, duration: audioRef.current.duration, buzz: Date.now() },
     );
 
     props.setReadingMode('waitforans');
   };
 
+
   return (
     <div className="question-speaker">
 
-      {(props.readingMode === 'waitforstrt') && <button type="button" onClick={play}>Play</button>}
-      {(props.readingMode === 'readactive') && <button type="button" onClick={buzz}>Buzz</button>}
+      {(isLoading === "loading") && <span className='loading-header'>
+        <i class="fa-solid fa-spinner fa-spin"></i>
+      </span>}
+      {(props.readingMode === 'waitforstrt' && isLoading === "ready") && <button type="button" onClick={play}>Play</button>}
```

Comment:
Could you make this simpler by checking if` props.readingMode === 'waitforstrt' && isLoading` your button will say "play" and then "buzz" otherwise? as oppposed to two buttons